src/main/java/com/aether/blocks/SpreadableAetherBlock.java

```diff
@@ -15,7 +15,7 @@ protected SpreadableAetherBlock(AbstractBlock.Settings settings) {
         super(settings);
     }
 
-    private static boolean canSurvive(BlockState state, WorldView worldView, BlockPos pos) {
+    private static boolean canBeGrass(BlockState state, WorldView worldView, BlockPos pos) {
```

Comment:
These may be artifacts from the mapping migration. TODO: Verify this before merge.